src/components/SplashScreen.tsx

```diff
@@ -2,9 +2,13 @@ import React from 'react';
 import styled from '@emotion/styled';
 import { css, keyframes, SerializedStyles } from '@emotion/core';
 
-export const SplashScreen: React.FC = () => (
+type Props = {
+  handleSplash: () => void;
```

Comment:
```suggestion
  onSplashEnd: () => void;
```

prop에는 on으로 시작, 어떤 이벤트에 대한 핸들러인지 명확하게 표현하면 좋을 듯!

Reply:
핡 완전 맞는 말.. 앞으로 이런 실수 없도록 할게유